scripts/gh_pr.sh

```
@@ -0,0 +1,149 @@
+    curl -s "$API_URL/pulls/$pr_id/files" | jq -r '.[] | "  \(if .status == "modified" then "M" else "A" end) \(.filename)"'
+}
+
+function diff_pr() {
+    local pr_id=$1
+    if [ -z "$pr_id" ]; then
+        echo "Usage: $0 diff <pr_number>"
+        exit 1
+    fi
+
+    echo -e "${BOLD}Fetching diff for PR #$pr_id...${NC}"
+    git fetch origin pull/$pr_id/head > /dev/null 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)